src/builders/indexable-author-builder.php

```diff
@@ -48,6 +54,7 @@ public function build( $user_id, Indexable $indexable ) {
 		$indexable->permalink              = \get_author_posts_url( $user_id );
 		$indexable->title                  = $meta_data['wpseo_title'];
 		$indexable->description            = $meta_data['wpseo_metadesc'];
+		$indexable->breadcrumb_title       = $this->options->get( 'breadcrumbs-archiveprefix' );
```

Comment:
Setting this here means that if this option value changes then all author indexables will have to be updated and the author watcher also needs to be changed to record changes to this option.

Instead I think the best approach would be to not set this value in the builder but rather to use what you've already suggested in https://github.com/Yoast/wordpress-seo/pull/15104/files#diff-87ec924042126a942ade18a29d1653daR249 ands set it in the breadcrumbs generator.

Reply:
Author is reflected in database when that user is updated.

Reply:
Yes breadcrumb generator is not using the value we save here. I just added this so that may be in future it can be used and make this consistent with other builders.